content/docs/rfcs/73/README.md

```diff
@@ -0,0 +1,161 @@
+
+This is useful when the prospective user does not have access to funds on the network that [32/RLN-V1](/spec/32) is deployed on.
+
+## Wire Format Specification 
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, 
+“NOT RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+The wire formats are encoded using protobuf.
+The wire formats that SHOULD be used by implementors include two message types, 
+`RequestToJoin` and `JoinResponse`.
+
+### Request to Join
+The `RequestToJoin` message MUST be used by users joining a membership set with assistance from the application or 
+other users.
+The message should include `spending_public_key` and 
+a `viewing_public_key` that is generated by the member’s public key.
+
+The protobuf message description is:
+
+```protobuf
+
+syntax = "proto3";
+
+message RequestToJoin {
+  // The Lamport timestamp of the request  
+  uint64 clock = 1;
+
+  // The ENS name of the requester
+  string ens_name = 2;
+
+  // The chat id requested to join
+  string pubsub_topic = 3;
+
+  // The public key of the application
+  bytes app_key = 4;
+
+  // The display name of the requester
+  string display_name = 5;
+
+  // The spending public key of the requester
+  bytes spending_public_key = 6;
+
+  // The viewing public key of the requester
+  bytes viewing_public_key = 7;
+}
+
+```
+
+The `spending_public_key` and
+`viewing_public_key` MUST be included in the message.
+Both attributes create a stealth meta-address,
+as described in [ERC-5564](https://eips.ethereum.org/EIPS/eip-5564).
+- both SHOULD be a 32-byte hash.
+- both SHOULD be derived from the community member's public key.
+
+### Generate Stealth Address
+The application or user SHOULD generate a stealth address after the message is received.
+This address MUST be used to publish the requester's [32/RLN-V1](/spec/32) `identitycommitment` for the corresponding `app_key` memebership set.
+
+- The receiver MUST generate an `ephemeral_public_key` and `stealth_commitment`.
+This will be used to check the stealth address used to update the memebership set,
+and the user MUST be able to check with their `viewing_public_key`.
+
+### Response for Request to Join
+The `JoinResponse` message MUST be used by applications/users willing to service the requester.
+
+```protobuf
+
+syntax = "proto3";
+
+message JoinResponse {
+  // The Lamport timestamp of the request
+  uint64 clock = 1;
+
+  // The community description
+  CommunityDescription community = 2;
+
+  // If the request was accepted
+  bool accepted = 3;
+
+  // The grant offered by the community
+  bytes grant = 4;
+
+  // The community public key
+  bytes community_id = 5;
+
+  // The stealth commitment for the requester
+  bytes stealth_commitment = 6;
+
+  // The ephemeral public key used to generate the commitment
+  bytes ephemeral_public_key = 7;
+}
```

Comment:
```suggestion
  // If the request was accepted
  bool accepted = 3;

  // The grant offered by the community
  bytes grant = 4;

  // The stealth commitment for the requester
  bytes stealth_commitment = 6;

  // The ephemeral public key used to generate the commitment
  bytes ephemeral_public_key = 7;
}
```

Removing everything that seems to be Status community related.

What is `grant` ?

for `accepted`, maybe best to have a `status_code` similar to other protocols: https://github.com/vacp2p/rfc/blob/84638042ce10b8416918056c93ca32bbd13556c9/content/docs/rfcs/12/README.md?plain=1#L97